feat: add autocomplete (#579)
jina-ai/now: docs/user-guides/production.md

Your data is automatically loaded from the S3 bucket. It will be temporary stored on EC2 machines while processing the data
but it will not be stored.

## Auto-complete suggestions

> For aplications with Text as a modality input, we provide `autocomplete` feature. This feature learns slowly from your users's search requests

Reviewer: ```suggestion
For applications with Text as input modality, we provide the `autocomplete` feature. This feature learns slowly from the user's search requests
```

Author: Thanks, applied

Your data is automatically loaded from the S3 bucket. It will be temporary stored on EC2 machines while processing the data
but it will not be stored.

## Auto-complete suggestions

> For applications with Text as input modality, we provide the `autocomplete` feature. This feature learns slowly from the user's search requests

Reviewer 2: text is lower case
